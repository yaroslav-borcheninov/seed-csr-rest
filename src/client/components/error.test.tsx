import React from "react"
import { shallow } from "enzyme"

import Error from "./error"

describe("Error", () => {
  it("should render", () => {
    expect(shallow(<Error />)).toMatchSnapshot()
  })
})
