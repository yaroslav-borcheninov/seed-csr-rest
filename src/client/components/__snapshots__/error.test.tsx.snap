// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render 1`] = `
<div>
  Error...
</div>
`;
